src/main/kotlin/org/jitsi/nlj/rtp/bandwidthestimation/AbsSendTimeBandwidthEstimator.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright @ 2019 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.nlj.rtp.bandwidthestimation
+
+import java.time.Duration
+import java.time.Instant
+import org.jitsi.nlj.util.Bandwidth
+import org.jitsi.nlj.util.DataSize
+import org.jitsi.nlj.util.bps
+import org.jitsi.nlj.util.createChildLogger
+import org.jitsi.nlj.util.kbps
+import org.jitsi.nlj.util.mbps
+import org.jitsi.nlj.util.observableWhenChanged
+import org.jitsi.utils.logging.DiagnosticContext
+import org.jitsi.utils.logging2.Logger
+import org.jitsi_modified.impl.neomedia.rtp.remotebitrateestimator.RemoteBitrateEstimatorAbsSendTime
+
+private val defaultInitBw: Bandwidth = 2.5.mbps
+private val defaultMinBw: Bandwidth = 30.kbps
+private val defaultMaxBw: Bandwidth = 20.mbps
+
+/**
+ * Just the delay-based part of Google CC.
+ */
+class AbsSendTimeBandwidthEstimator(diagnosticContext: DiagnosticContext, parentLogger: Logger) :
```

Comment:
This name is a bit confusing given there's also the `RemoteBitrateEstimatorAbsSendTime` class.  This class is basically a `BandwidthEstimator` wrapper around `RemoteBitrateEstimatorAbsSendTime`?